swarm/network/stream/stream.go

```diff
@@ -571,11 +554,12 @@ func (c *client) nextBatch(from uint64) (nextFrom uint64, nextTo uint64) {
 	if nextTo == 0 {
 		nextTo = c.sessionAt
 	}
+	log.Debug("client.nextBatch", "from", from, "nextFrom", nextFrom, "nextTo", nextTo)
 	return
 }
 
 func (c *client) batchDone(p *Peer, req *OfferedHashesMsg, hashes []byte) error {
-	if tf := c.BatchDone(req.Stream, req.From, hashes, req.Root); tf != nil {
+	/*if tf := c.BatchDone(req.Stream, req.From, hashes, req.Root); tf != nil {
```

Comment:
@justelad I would remove this rather than just comment. This is unnecessary indirection.